src/EFCore.PG/Storage/Internal/Mapping/NpgsqlListTypeMapping.cs

```diff
@@ -29,19 +29,23 @@
 using Microsoft.EntityFrameworkCore.Storage;
 using System.Collections.Generic;
 using System.Diagnostics;
+using JetBrains.Annotations;
 using Microsoft.EntityFrameworkCore.ChangeTracking;
 
 namespace Npgsql.EntityFrameworkCore.PostgreSQL.Storage.Internal.Mapping
 {
     /// <summary>
     /// Maps PostgreSQL arrays to <see cref="List{T}"/>.
     /// </summary>
+    /// <remarks>
+    /// Note that mapping PostgreSQL arrays to .NET arrays is also supported via <see cref="NpgsqlArrayTypeMapping"/>.
+    /// </remarks>
     public class NpgsqlListTypeMapping : RelationalTypeMapping
     {
         /// <summary>
         /// The CLR type of the list items.
         /// </summary>
-        public RelationalTypeMapping ElementMapping { get; }
+        [NotNull] readonly RelationalTypeMapping _elementMapping;
```

Comment:
Is there a reason to make it a field?

Reply:
Agree with @YohDeadfall, it's true that we may not be accessing the element mapping from the outside, but it seems like a good think to have on the public API of the class.

Reply:
Just the result of a series of ReSharper-refactors (private property -> private field -> readonly field).

I'll revert.